metagraph/src/cli/server.cpp

```diff
@@ -130,6 +130,8 @@ std::string process_search_request(const std::string &received_message,
     config.alignment_max_nodes_per_seq_char = json.get(
         "max_num_nodes_per_seq_char",
         config.alignment_max_nodes_per_seq_char).asDouble();
+    config.alignment_max_ram = json.get("max_ram_per_alignment",
+                                        config.alignment_max_ram).asDouble();
```

Comment:
this is not safe, don't pass it with api

Reply:
Good point. We could then write in the manual that the server has this limit and that the CLI version should be used if one needs more RAM for their alignment.